jekyll/_style/index.md

```diff
@@ -24,5 +24,4 @@ For further information on markdown and asciidoc syntax refer to the [markdown d
     <li><a href="{{ site.baseurl }}/style/style-screenshots/">Using screenshots</a></li>
     <li><a href="{{ site.baseurl }}/style/style-images/">Using images</a></li>
   <ul>
-</div>
```

Comment:
Why did we remove the closing div? There is an opening div above the list.

Reply:
because with it in it was appearing on the page - something to do with the html in the layout file I think